iocage/lib/Config/Jail/Defaults.py

```diff
@@ -86,7 +86,8 @@ class JailConfigDefaults(iocage.lib.Config.Jail.BaseConfig.BaseConfig):
         "ip6_saddrsel": 1,
         "ip6_addr": None,
         "resolver": "/etc/resolv.conf",
-        "host_domainname": "none",
+        "host_hostname": None,
+        "host_domainname": None,
```

Comment:
i'm wondering if we shouldn't set 'local' **here**

Reply:
Fine by me. I'd love to see a PR. @skarekrow what's your opinion?